dashbuilder-client/dashbuilder-displayer-client/src/main/java/org/dashbuilder/displayer/client/DisplayerCoordinator.java

```diff
@@ -208,6 +208,15 @@ public void onDataLookup(Displayer displayer) {
         }
 
         @Override
+        public void onDataLoaded(Displayer displayer) {
+            for (Displayer other : displayerList) {
```

Comment:
Hey @dgutierr , I would recommend being used to  Java8 functional interfaces and the Streams API as soon as possible and try to use it on the new stuff, so for doing kind an incremental replacement..